README.md

```diff
@@ -235,6 +222,22 @@ print(naval_chat_bot.chat("what did the author say about happiness?"))
 # answer: The author, Naval Ravikant, believes that happiness is a choice you make and a skill you develop. He compares the mind to the body, stating that just as the body can be molded and changed, so can the mind. He emphasizes the importance of being present in the moment and not getting caught up in regrets of the past or worries about the future. By being present and grateful for where you are, you can experience true happiness.
 ```
 
+### Stream Response
+
+- You can add config to your query method to stream responses like ChatGPT does. You would require a downstream handler to render the chunk in your desirable format. Currently only supports OpenAI model.
+
+- To use this, instantiate a `QueryConfig` or `ChatConfig` object with `stream_response=True`. Then pass it to the `.chat()` or `.query()` method. The following example iterates through the chunks and prints them as they appear.
+
+```python
+app = App()
+query_config = QueryConfig(stream_response = True)
+resp = naval_chat_bot.query("What unique capacity does Naval argue humans possess when it comes to understanding explanations or concepts?", query_config)
+
+for chunk in resp:
```

Comment:
should we give a downstream handler? assuming that a lot of users will use streaming response, so why not there is a function which when called gives the streaming response output.
@cachho @aaishikdutta

Reply:
not everyone's going to want to just print it. I haven't looked into how I can get route the stream through flask, but that's my usecase for instance, so no, I'm fine with it how it is.

Streaming is for people that know a little. Not for private users that want to play around. If you want to use it professionally you should be able to copy two lines from the readme.

Reply:
Okay, then for now lets leave this.